src/commandsHandler/get-delegate-data.ts

```diff
@@ -3,6 +3,22 @@
 import { isEthAddress } from '../utils/is-eth-address';
 import { api } from '../api/index';
 
+const createMessageInfo = (
```

Comment:
Name of this function does not tell me what she is doing
Use non arrow function